string box_type
float32 x
float32 y
float32 height
float32 width
string color